contracts/test/foundry/FleekERC721/AccessPoints/AccessPointsAutoApprovalOff.t.sol

```diff
@@ -33,32 +35,22 @@ contract Test_FleekERC721_AccessPoint is Test_FleekERC721_Base {
         string memory accessPointName = "accesspoint.com";
         CuT.addAccessPoint(tokenId, accessPointName);
 
-        assertAccessPointJSON(accessPointName, "0", "0", "false", "false", deployer);
+        assertAccessPointJSON(accessPointName, "0", "0", "false", "false", deployer, "0");
     }
 
     function test_removeAccessPoint() public {
         string memory accessPointName = "accesspoint.com";
         CuT.addAccessPoint(tokenId, accessPointName);
         CuT.removeAccessPoint(accessPointName);
 
-        expectRevertWithInvalidAP();
-        CuT.getAccessPointJSON(accessPointName);
+        assertAccessPointJSON(accessPointName, "0", "0", "false", "false", deployer, "3");
     }
 
-    function test_cannotRemoveNonexistentAccessPoint() public {
+    function test_cannotRemoveNonExistentAccessPoint() public {
         expectRevertWithInvalidAP();
         CuT.removeAccessPoint("accesspoint.com");
     }
 
-    function test_cannotTwiceRemoveAccessPoint() public {
```

Comment:
Isn't this test still valid?

Reply:
Bump

Reply:
No. We change the `status` to `REMOVED` after removing the AP and keep the object. We discussed keeping history when we were discussing removing the mapping in the `App` struct.